src/openms/source/FORMAT/HANDLERS/MzIdentMLDOMHandler.cpp

```diff
@@ -173,8 +173,9 @@ namespace OpenMS
           throw (runtime_error("Path file_name does not exist, or path is an empty string."));
         else if (errno == ENOTDIR)
           throw (runtime_error("A component of the path is not a directory."));
-        else if (errno == ELOOP)
-          throw (runtime_error("Too many symbolic links encountered while traversing the path."));
+        // On MSVC 2008, the ELOOP constant is not declared and thus introduces a compile error
```

Comment:
what happens with other errors?!
should there be a default, i.e. 
```
}
else
{
  throw ...
}
```

maybe not mission critical here, since I understand that the DOM parser will be replaced soon.